### DISCUSSION TITLE
Trash-cli support, or custom action support so I can integrate trash-cli with delete action?

### DISCUSSION THREAD

**Discussion by linrongbin16** [nvim-neo-tree/neo-tree.nvim#1037] [Q&A] [answered]
Hi, I'm using neo-tree.nvim under Windows 10, sometimes I feel trash-cli is much better for Windows with 2 reasons:
1. `rm` command is quite linux way, I need to install `coreutils` with [scoop](https://scoop.sh/#/apps?q=coreutils) to provide linux compatible commands.
2. Moving deleted things to recycle bin is much more fault-tolerant for a developer.

**Comment by miversen33** [comment, marked as answer]
#202 #860

**Reply by linrongbin16** [reply]
thanks for reply
